src/p_enemy.c

```diff
@@ -439,6 +439,7 @@ static boolean P_Move(mobj_t *actor, boolean dropoff) // killough 9/12/98
       for (good = false; numspechit--; )
         if (P_UseSpecialLine(actor, spechit[numspechit], 0, false))
 	  good |= spechit[numspechit] == blockline ? 1 : 2;
+      numspechit = 0;
```

Comment:
Should these lines get a comment? Neither DSDA-Doom nor Crispy have similar code.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1751/commits/68365e615b0adf96c93c593fe86c772db5ad2c26